src/core/currency/wallet/currency-wallet-files.js

```diff
@@ -45,6 +45,7 @@ export type TransactionFile = {
     currency: string,
     tag?: string
   }>,
+  secret?: string,
```

Comment:
Can you change to `txSecret`. Thats what we had decided and its easier to search for.